.idea/codeStyles/Project.xml

```diff
@@ -20,9 +20,6 @@
       <option name="NAME_COUNT_TO_USE_STAR_IMPORT_FOR_MEMBERS" value="50" />
       <option name="ALLOW_TRAILING_COMMA" value="true" />
     </JetCodeStyleSettings>
-    <XML>
```

Comment:
👀

Reply:
Oof. I missed that in the list of changes, sorry. I hate it when the IDE does stuff randomly.

Reply:
No worries 😅